### DISCUSSION TITLE
Is there a way for breadcrumbs to show the last N pages the reader clicked?

### DISCUSSION THREAD

**Discussion by ajeanmahoney** [requarks/wiki#6842] [Help / Questions]
It looks like breadcrumbs are constructed by specifying a folder structure (e.g., /universe/planets/earth structure in the [docs](https://docs.requarks.io/en/guide/structure)). Is there a way to show, say, the last three or four links the reader actually clicked? So instead of "home / install / config", the breadcrumbs show "Home, User Guide, Configuration, Logging"?

**Comment by NGPixel** [comment]
No, although that would be very similar to right-clicking your browser's back button? The breadcrumbs are meant to go back up in hierarchy, not in history. This could confuse users with potentially unrelated links next to each other.
